net-mgmt/zabbix-agent/Makefile

```diff
@@ -1,8 +1,11 @@
 PLUGIN_NAME=		zabbix-agent
-PLUGIN_VERSION=		1.14
+PLUGIN_VERSION=		1.15
 PLUGIN_COMMENT=		Zabbix monitoring agent
 PLUGIN_MAINTAINER=	opnsense@moov.de
-PLUGIN_VARIANTS=	zabbix7 zabbix6 zabbix64 zabbix5
+PLUGIN_VARIANTS=	zabbix72 zabbix7 zabbix6 zabbix64 zabbix5
```

Comment:
I would prefer the LTS in the first spot, otherwise the default switches to 7.2 like it is now